### ISSUE TITLE
[Bug?] Handy Lockpick feature doesn't seem to work on some boxes

### ISSUE THREAD

**Issue by zeenk** [Peterodox/Plumber#48]
![2024-12-15_03-17-07](https://github.com/user-attachments/assets/4f36de55-0f50-4c40-94fd-1f2bed152e4e)
![2024-12-15_03-30-13](https://github.com/user-attachments/assets/e26c0f5b-e588-46ef-8e58-0302d080ac0a)

This feature seem to be broken on Retail, I haven't tested any other wow versions.
I first thought it was just not compatible with Baganator but I also tried with only Plumber enabled and same results, no tooltip and nothing happens when you try to right click to unlock it

**Comment by zeenk** [comment]
After some further testing I think it's because of the BoE label on this specific lockbox, I tried with an older one in trade window and that one worked

But it works if I use the pick lock spell on the BoE box in trade window, just can't right click them

**Comment by Peterodox** [comment]
It failed to work because I only checked the second line on the tooltip, looking for "Locked", to minimize unnecessary processing power. I'll start scanning the third line in the next update.
Why is there Binds when equipped on this thing 🤣

**Comment by zeenk** [comment]
Yeah good question.. can't sell these on AH anyway so makes no sense to have it labeled as BoE lol

**Comment by zeenk** [comment]
I saw in the 1.5.1 changelog that the fix was added but doesn't seem to work sadly, tried with only Plumber enabled
![2024-12-18_00-47-32](https://github.com/user-attachments/assets/e7368267-b35b-480c-8096-cbfad533e169)

**Comment by zeenk** [comment]
Just saw a different screenshot of it, it can have different ranks too? 🤨 
![image](https://github.com/user-attachments/assets/d1eac124-7f5d-4efa-a587-59ec1b2f7ef2)

**Comment by Peterodox** [comment]
This item is cursed.

There was a exploit in the beginning of this expansion where you had a chance to get an item with lv.80 stats but equippable at lv.70. That's why people were buying this stuff for thousands of gold.

I'm going fine tune my code for this edge case, so it'll scan all tooltip lines (from the second one of course) when you hover over this lockbox.

**Comment by zeenk** [comment]
Oh, this seem to be working now so closing this issue again.
![2025-01-06_22-41-59](https://github.com/user-attachments/assets/7956b3ea-7c1b-4126-959d-0c2fa5a8a2f8)
